CarRentals/CarRentalsWebAPI/Controllers/BrandsController.cs

```diff
@@ -0,0 +1,103 @@
+﻿using CarRentalsWebAPI.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarRentalsWebAPI.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
do you understand how does this route work? and what the `[controller]` placeholder represents?

Reply:
I know this is the path we use for Get methods where api / [Controller] = api / Classname without "Controller". When we use GetById, the [HttpGet ({id})] tag takes, also the Id that is specified behind the tag.

Reply:
it's not about the GET but ALL the endpoints listed inside the controller, there are some conventions that are usually followed and mechanisms we have to alter de default behaviour.
as you correctly answered, in this case the [Controller] represents `brands` meaning we have `/api/brands` but we can still have more in our routes: imagine we have the following `[HttpGet("{id}/cars")]`, this mean our path will look like `/api/brands/{id}/cars`, obviously this is not only for GET but for any verb. 
there's a lot of info and best practices around this paths and how they should be built, but not our primary target right now, just wanted to be sure you understand this and know you can remove the placeholder and use whatever route you want like